flowermd/modules/surface_wetting/surface_wetting.py

```diff
@@ -95,21 +95,27 @@ def run_droplet(
 
         """
         # Shrink down to high density
+        target_box = get_target_box_mass_density(
+            density=shrink_density * (u.g / (u.cm**3)), mass=self.mass.to("g")
```

Comment:
Maybe we need an if statement here. Maybe someone passes in other density units for `final_density` and `shrink_density` and we don't always want to use g/cm^3

Reply:
I guess the behavior here should basically be the same as what we do in `Pack` now.

Reply:
Agree. same as in `Pack` we could first check if the units exists and if not assume the default is `g/cm^3`